ui/.template-lintrc.js

```diff
@@ -47,7 +47,7 @@ module.exports = {
     },
     'require-input-label': 'off',
   },
-  ignore: ['lib/story-md', 'tests/**'],
```

Comment:
`lib/story-md` is old, right? i didn't see it anywhere

Reply:
Hmm I'm not sure what `lib/story-md` is but I did a quick search of it in our codebase! It looks like it's being used in one of the scripts (`scripts/gen-story-md.js`). Could we also remove the scripts if `lib/story-md` is no longer needed?

Reply:
yeah, it was used with storybook. [See PR here.](https://github.com/hashicorp/vault/pull/15074/files) I think the reason I didn't remove gen-story-md is just in case scenario.... but I'd say it's safe to remove as well as the lib/story-md. Maybe in a separate PR that cleans up the left over files?